addons/account/models/company.py

```diff
@@ -190,6 +190,10 @@ class ResCompany(models.Model):
             ('out_and_in_invoices', 'Customer Invoices and Vendor Bills')],
         string="Quick encoding")
 
+    # Internally set by the localization packages
+    # False if the localization want to hide the taxes column
+    show_taxes = fields.Boolean(store=False, default=True, check_company=True)
```

Comment:
Check company doesn't work apparently, did not know that parameter though

Reply:
It's not useful here, I'll remove it